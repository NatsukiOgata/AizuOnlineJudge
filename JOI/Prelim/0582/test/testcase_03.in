55 49 83
87 79 87
57 82 42
95 91 88
73 12 75
10 95 86
29 61 71
24 14 29
63 35 40
48 35 28
4 25 26
98 87 57
39 24 43
39 93 81
42 34 43
42 50 74
67 62 33
20 35 45
80 18 82
8 17 15
34 30 16
53 60 36
31 100 80
94 13 82
50 71 62
34 30 16
74 30 71
69 33 95
16 9 19
58 71 36
67 59 33
77 91 27
98 30 92
41 76 53
21 20 29
48 64 51
92 53 73
49 64 23
59 56 65
65 62 48
89 80 39
44 13 55
73 50 91
55 45 54
85 79 24
15 8 17
85 54 47
84 81 37
28 69 58
56 79 90
43 54 16
60 32 76
94 99 30
90 79 87
18 18 2
9 40 41
61 33 60
97 62 59
71 88 56
81 70 89
47 97 53
23 20 28
27 76 72
51 90 98
98 56 46
77 30 77
15 46 27
24 40 32
55 57 36
89 86 19
45 25 23
46 53 62
48 67 63
50 100 86
80 55 34
26 23 67
45 44 27
25 58 45
21 20 29
99 34 76
20 16 12
35 33 42
56 75 68
67 44 27
82 51 37
64 78 94
19 6 19
30 50 2
14 80 76
32 68 60
51 59 24
45 11 37
43 44 40
50 27 26
47 6 52
76 18 79
7 4 55
83 42 16
46 94 58
50 44 20
70 87 54
58 63 42
98 43 75
59 81 84
56 86 97
29 20 21
59 22 72
87 47 58
90 42 90
27 36 45
49 60 71
56 33 69
75 69 77
46 72 93
24 70 74
18 69 73
11 94 90
46 87 60
52 28 44
11 62 55
25 13 25
14 62 58
69 77 48
41 31 28
54 98 58
40 99 62
90 22 97
40 56 78
35 85 91
26 63 70
70 65 61
39 88 51
7 82 76
73 45 36
90 100 16
16 27 16
66 78 60
60 65 83
11 97 87
92 96 82
62 34 89
29 87 100
51 79 92
74 32 45
67 63 30
58 31 35
80 64 48
89 80 39
33 56 65
23 80 59
72 35 56
73 44 65
90 85 38
65 43 29
12 93 91
80 27 97
62 37 82
27 75 56
84 36 64
30 34 16
23 39 26
86 87 55
53 35 85
71 47 99
14 89 97
70 62 51
24 70 74
63 74 55
53 48 27
22 43 62
39 26 24
60 87 32
22 52 33
65 72 58
70 15 70
44 37 42
77 51 33
84 61 28
46 59 85
16 58 50
36 85 77
98 83 47
5 87 84
59 12 64
70 74 24
70 100 76
36 49 77
72 28 75
17 29 44
44 31 66
92 55 62
45 28 53
37 19 24
58 77 62
41 39 3
66 43 35
58 42 40
55 72 58
10 28 14
84 63 26
