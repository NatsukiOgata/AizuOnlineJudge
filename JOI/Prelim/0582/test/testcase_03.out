66 8 25 33
